### ISSUE TITLE
Can't build servo - "Failed to run custom build command for `harfbuzz"

### ISSUE THREAD

**Issue by kevincannon** [servo/servo#4732]
This is the issue i'm getting. I've searched around, but not found anything.

Anyone got any ideas. Running latest version of OSX Yosemite. 10.10.1

```
/Users/kevin.cannon/.cargo/git/checkouts/gcc-rs-769a83e2edfc991e/servo/src/lib.rs:1:12: 1:18 warning: feature has been added to Rust, directive not necessary
/Users/kevin.cannon/.cargo/git/checkouts/gcc-rs-769a83e2edfc991e/servo/src/lib.rs:1 #![feature(if_let)]
                                                                                               ^~~~~~
   Compiling lazy_static v0.1.0 (https://github.com/Kimundi/lazy-static.rs#76f06e4f)
Build failed, waiting for other jobs to finish...
Failed to run custom build command for `harfbuzz v0.1.0 (https://github.com/servo/rust-harfbuzz#59b5b180)`
Process didn't exit successfully: `make -f makefile.cargo` (status=2)
--- stderr
makefile.cargo:75: warning: overriding commands for target `/Users/kevin.cannon/Documents/05.'
makefile.cargo:72: warning: ignoring old commands for target `/Users/kevin.cannon/Documents/05.'
make: *** No rule to make target `harfbuzz/src/%.cc', needed by `/Users/kevin.cannon/Documents/05.'.  Stop.
```

**Comment by mbrubeck** [comment]
> `/Users/kevin.cannon/Documents/05.`

Is this an actual filename on your system, or is the Servo code maybe inside a folder with a space in its name, which one of our makefiles is not escaping correctly?

**Comment by kevincannon** [comment]
Bingo! That was it, I can compile it now.

After a little bit of testing, it seems the build script is failing when a folder has a . in the name. In my company, our user accounts are created automatically in firstname.lastname, so if I move it out of my user folder it compiles.

**Comment by mbrubeck** [comment]
Should be fixed by servo/rust-harfbuzz#53.
